config/application.yml.default

```diff
@@ -146,6 +146,8 @@ irs_attempt_api_auth_tokens: ''
 irs_attempt_api_csp_id: 'LOGIN.gov'
 irs_attempt_api_enabled: false
 irs_attempt_api_aws_s3_enabled: false
+irs_attempt_api_aws_s3_stream_enabled: false
+irs_attempt_api_aws_s3_stream_buffer_size: 100_000
```

Comment:
I'm not going on any scientific data, but I would look at making this at least 1MB and maybe much more. We don't want it to be multiple GBs, but each read is going to add latency. If we figure production files might be something like 1GB (SWAG), this would result in 10,000 requests. Call it 50ms per read (SWAG) and we're at 500 seconds. 16MB would drop us to 63 requests and ~3 seconds.

We don't need to sweat over getting this just right since we can tune it in our config, and I should reiterate that I'm going by nothing but gut feeling.

Reply:
Done. the default is now `16_777_216` or 16MB.